docs/userGuide/redirectingToCustom404Page.md

```diff
@@ -0,0 +1,23 @@
+{% set title = "Redirecting to Custom 404 Page" %}
+{% set filename = "redirectingToCustom404Page" %}
+
+<frontmatter>
+  title: "User Guide: {{ title }}"
+  layout: userGuide.md
+</frontmatter>
+
+<span id="link" class="d-none">
+<md>[_User Guide → {{ title }}_]({{ filename }}.html)</md>
+</span>
+
+# {{ title }}
+
+<div class="lead" id="overview">
+Most popular static hosting service such as Netlify supports custom routing to a custom 404 html page. A default 404.md page is provided, and you can customise it to create your own 404 page.
```

Comment:
Perhaps more details/hyperlinks can be included. You can reference the links given in the original issue to see the documentation for those static hosting services such as (e.g. [in Gatsby](https://www.gatsbyjs.com/docs/how-to/adding-common-features/add-404-page/), [Hugo](https://gohugo.io/templates/404/), [Jekyll](https://jekyllrb.com/tutorials/custom-404-page/), [Netlify](https://docs.netlify.com/routing/redirects/redirect-options/#custom-404-page-handling)) 

Some nits:

```suggestion
Most popular static hosting service such as Netlify supports custom routing to a custom 404 HTML page. A default `404.md` file is provided, and you can customize it to create your own 404 page.
```